src/platforms/linux/linuxpingsender.cpp

```diff
@@ -157,22 +157,22 @@ void LinuxPingSender::rawSocketReady() {
   unsigned char data[2048];
   int rc = recvfrom(m_socket, data, sizeof(data), MSG_DONTWAIT, NULL, &slen);
   if (rc <= 0) {
-    logger.log() << "recvfrom failed:" << strerror(errno);
+    logger.error() << "recvfrom failed:" << strerror(errno);
     return;
   }
 
   // Check the IP header
   const struct iphdr* ip = (struct iphdr*)data;
   int iphdrlen = ip->ihl * 4;
   if (rc < iphdrlen || iphdrlen < (int)sizeof(struct iphdr)) {
-    logger.log() << "malformed IP packet:" << strerror(errno);
+    logger.error() << "malformed IP packet:" << strerror(errno);
     return;
   }
 
   // Check the ICMP packet
   struct icmphdr packet;
   if (inetChecksum(data + iphdrlen, rc - iphdrlen) != 0) {
-    logger.log() << "invalid checksum";
+    logger.error() << "invalid checksum";
```

Comment:
This should be reduced to warning() or maybe even debug(). It's an unlikely event but could occur due to network conditions.